packages/sdk/src/lib/IExecDataProtectorModule.ts

```diff
@@ -0,0 +1,46 @@
+import { Eip1193Provider } from 'ethers';
+import { GraphQLClient } from 'graphql-request';
+import { IExec } from 'iexec';
+import {
+  DEFAULT_CONTRACT_ADDRESS,
+  DEFAULT_IEXEC_IPFS_NODE,
+  DEFAULT_IPFS_GATEWAY,
+  DEFAULT_SUBGRAPH_URL,
+} from '../config/config.js';
+import { AddressOrENS, Web3SignerProvider } from './types/commonTypes.js';
+import { DataProtectorConfigOptions } from './types/dataProtectorTypes.js';
```

Comment:
Can we get these types from `./types/index.js`?
```suggestion
import {
  AddressOrENS,
  DataProtectorConfigOptions,
  Web3SignerProvider,
} from './types/index.js';
```